### PR TITLE
Welcome page skeleton

### PR DESCRIPTION
**Pull request by Jeffery05**
## Notion ticket link
[Welcome Page Skeleton](https://www.notion.so/uwblueprintexecs/7e51fde5f22d448b8226f2cfa4dd93d5?v=b2dc0dba06574651a8b6063bdd0d74e0&p=126941418e264b42a710e71ed1484a30&pm=s)

## Implementation description
* Created a new route for "/welcome"
* Added a new component WelcomePage
* Added three buttons and a handleButtonClick() function to redirect to '/login?role=<role>' once a button is clicked

## Steps to test
1. Ensure the Welcome page exists
2. Ensure the routing to the Welcome Page works ('/welcome')
3. Ensure you are correctly redirected to '/login?role=<role>' with the correct role (depending on the button you pressed, e.g. '/login?role=administrator' for administrators)

## What should reviewers focus on?
* Double-check that this is a new entry in the history stack (e.g. clicking the browser ← button should bring me back to the Welcome page)

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (carolynzhang18) [inline yarn.lock:1]: We can delete this `yarn.lock` file - I'm guessing you ran a `yarn` command from the root directory by accident at some point :D

**Review by carolynzhang18 — COMMENTED** [review]
Nice work!! Everything works correctly! 🤩 2 tiny comments but otherwise LGTM

**Review by carolynzhang18 — APPROVED** [review]
🚢 it!!

_(after 1 more dev approves)_

**Review by 98ZhaoJeffrey — APPROVED** [review]
W commit

**Review by ji-mmyliu — APPROVED** [review]
LGTM :+1:

**Comment by ji-mmyliu** [comment]
> LGTM 👍

Oops, it got merged as I was reviewing it LOL

**Pull request merged**